content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md

```diff
@@ -97,24 +97,6 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `check.cache_hit` | boolean | Indicates whether Mixer check call hits local cache. | |
 | `quota.cache_hit` | boolean | Indicates whether Mixer quota call hits local cache. | |
 
-## Deprecated attributes
+## Operate timestamp attributes
```

Comment:
# How to use timestamps

Reply:
I changed it to how to use timestamp attributes.